src/main/features/response/routes/response-type.ts

```diff
@@ -19,7 +20,9 @@ function renderView (form: Form<Response>, res: express.Response) {
 export default express.Router()
   .get(Paths.responseTypePage.uri,
     ErrorHandling.apply(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
-      renderView(new Form(res.locals.user.responseDraft.document.response), res)
+      const draft: ResponseDraft = res.locals.user.responseDraft.document
```

Comment:
no user being extracted here?

Reply:
I did it on purpose since sometimes a user variable is not needed directly because we just need a claim or one of the drafts. I decided not to extract user variable in such cases as my thinking was that we still get type information needed in given scope.

I am aware that it might still make us some issue when we refactor User or Draft classes but I was happy to take the risk to make code a bit nicer. I knew it might trigger questions so I am open for conversation

Note: Also in the future I am thinking that claim or draft properties should be stored in `res.locals` map directly not as a sub property of user instance.

Reply:
I would prefer if we were consistent, it seems like adding 2 rules doing it this way which makes it harder to explain why something is like it is.
Is it a mistake or intentional?

Reply:
Possible approaches:

```
const draft: ResponseDraft = res.locals.user.responseDraft.document
renderView(new Form(draft.document.response), res)
```

or

```
const user: User = res.locals.user
const draft: ResponseDraft = user.responseDraft.document
renderView(new Form(draft.document.response), res)
```

or

```
const user: User = res.locals.user
renderView(new Form(user.responseDraft.document.document.response), res)
```

Reply:
I would prefer the third one

Reply:
or moving it off the user object

Reply:
Yes the third option is better, did a refactor in CCJ check and send route in last sprint